frontend/src/components/VHeader/VHeaderDesktop.vue

```diff
@@ -106,11 +108,19 @@ export default defineComponent({
       document.activeElement?.blur()
       updateSearchState()
     }
+
     const areFiltersDisabled = computed(
       () => !searchStore.searchTypeIsSupported
     )
 
-    const toggleSidebar = () => uiStore.toggleFilters()
+    const toggleSidebar = () => {
+      const state = isSidebarVisible?.value ? "closed" : "opened"
+      sendCustomEvent("TOGGLE_FILTER_SIDEBAR", {
+        mediaType: searchStore.searchType,
+        state,
```

Comment:
This object doesn't seem to match the type:
- `TOGGLE_FILTER_SIDEBAR` expects `toState` prop (instead of `state`)
- `state` is expected to be "open", not "opened"
I wonder why TS didn't catch this as errors?

Reply:
ah, thank you! i'll check what's going on with my local TS server for me to have missed this 🤦